### PR TITLE
FEATURE: Allow configuration of rabbit queue name

### PR DESCRIPTION
**Pull request by paxuclus**
If a queue with the same name exists in multiple RabbitMQ vhosts, using all of them would not be possible without configuring the queue name.

### CODE REVIEW

File: Classes/Queue/RabbitQueue.php [hunk 1]
```diff
@@ -95,6 +100,7 @@ public function __construct(string $name, array $options = [])
         // declare queue
         $queueOptions = $options['queueOptions'];
 
+        $this->queueName = isset($queueOptions['name']) ? $queueOptions['name'] : $this->name;
```
Comment (johannessteu) [inline Classes/Queue/RabbitQueue.php]:
```suggestion
        $this->queueName = isset($queueOptions['name']) ?? $this->name;
```

**Review by johannessteu — CHANGES_REQUESTED** [review]
Fix lint issues

File: Classes/Queue/RabbitQueue.php (hunk 1 above)
Comment (paxuclus) [reply Classes/Queue/RabbitQueue.php]: Whoops, done!

Comment (johannessteu) [reply Classes/Queue/RabbitQueue.php]: Thanks ❤️

**Pull request merged**